apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -689,6 +688,38 @@ def __len__(self) -> int:
         """Returns the number of rows in the dataframe. Same as ``df.count``."""
         return self.count
 
+    def delete_cells(
+        self,
+        coords: options.SparseDFCoords,
+        *,
+        value_filter: str | None = None,
+        platform_config: options.PlatformConfig | None = None,
+    ) -> None:
+        """Deletes cells at the specified coordinates.
+
+        Note: Deleting cells with an enumeration value does not effect the possible enumerations.
```

Comment:
given this helpful note, is it also worth noting that "shape" is unaffected in the NDArrays?